ICON/main.yaml

```diff
@@ -324,3 +324,31 @@ sources:
       project: ILUMInESS
       source_id: ICON-ESM
       simulation_id: ngc5004
+  ngc5004kerchunk:
```

Comment:
If the kerchunk-access is what we recommend (I guess it will be), then I'd propose to replace the old catalog entry with the kerchunk access. This will:
* confuse people less
* improve performance for everyone

Reply:
i would agree if it hadnt been said that there will be more data to come. kerchunk is only a static snapshot.